docs/annotations.md

```diff
@@ -1,7 +1,13 @@
 # Annotations
 
-Contour supports the following annotations.
+Annotations are used in Ingress Controllers to configure features that are not
+covered by the Kubernetes Ingress API.
 
+Some of the features that have been historically configured via annotations are
+supported as first-class features in Contour's [IngressRoute
+API](ingressroute.md), which provides a more robust configuration interface over
+annotations. However, Contour still supports a number of annotations on both the
```

Comment:
Huh, not sure why I was under the impression that we did. So currently there's no way in IngressRoutes to configure some of the things that are otherwise doable in Ingress + annotations? For example, request timeout, retry-on, etc.